tests/p2p/test_twin_tx.py

```diff
@@ -69,6 +71,9 @@ def test_twin_tx(self):
         self.assertEqual(meta2.voided_by, {tx2.hash})
         self.assertEqual(meta2.twins, [tx1.hash])
 
+        # The same as tx1 but with one output different, so it's not a twin
+        tx3 = add_new_double_spending(self.manager, tx=tx1)
```

Comment:
Does it already propagate the transaction? Maybe the `self.manager.propagate_tx(tx3)` is not necessary.

Reply:
Yes `add_new_double_spending` has that call, I'll remove the redundant call.

Reply:
Great! Thanks!